### DISCUSSION TITLE
Port for Vim

### DISCUSSION THREAD

**Discussion by shivesh01** [daltonmenezes/aura-theme#117] [Port request]
I think there is a need for vim plugin because vim is popular nowadays..

**Comment by ashishbinu** [comment]
I would like to work on this issue.

**Comment by shivesh01** [comment]
ashish, are you working on this? If you are not working, may I work on this?

**Comment by ashishbinu** [comment]
@shivesh01 yeah I am working on it. My exams are also going on that's why it is delayed a bit.

**Comment by patrickdeangelis** [comment]
@ashishbinu how can I help the development?

**Comment by kavinvalli** [comment]
How long will this and #28 take @ashishbinu? Looking forward to the port

**Comment by Arjun31415** [comment]
I have created a port, however it is still lacking as I am not very familiar with themeing in vim
